contentcuration/contentcuration/views/settings.py

```diff
@@ -182,8 +182,9 @@ def form_valid(self, form):
                 "channels": channels,
             },
         )
+
         send_mail(
-            "Kolibri Studio storage request",
+            f"Kolibri Studio storage request from {self.request.user}",
```

Comment:
@Abhishekzod007 This change is unrelated to the issue referenced in the PR description.

Reply:
I think it was a mistake by me